.github/workflows/update.yml

```diff
@@ -24,18 +24,17 @@ jobs:
     - name: Check out cve-feed-osv repo
```

Comment:
We don't need this step. `cve-feed-osv` is already checked out as below.
https://github.com/kubernetes-sigs/cve-feed-osv/blob/bfb4a9d002b09aebac0e89fea9ddb08122a03bf3/.github/workflows/update.yml#L16-L17